client/app/models/collection.js

```diff
@@ -7,6 +7,7 @@ export default DS.Model.extend({
     created_by: DS.belongsTo('user'),
     dateCreated: DS.attr('date'),
     dateUpdated: DS.attr('date'),
+    settings: DS.attr('object'),
```

Comment:
I see your point, I had to do this to make the settings work as an object throughout the application, but because we are not saving any settings yet it didn't cause any problems. We need to look into this in a bit more detail but that might be beyond the scope of the current product. I'll take a look after all else.